docs/source/hardware/remote_controllers.rst

```diff
@@ -1,3 +1,104 @@
 ==================
 Remote Controllers
 ==================
+
+.. meta::
+    :description lang=en: Hardware page for the quadrotor's remote control transmitter.
+
+The quadrotors require a standard remote control transmitter to 
+
+* arm the flight controller, 
+* select flight control modes, 
+* fly manually, and
+* perform emergency landings. 
```

Comment:
I'd make these start with a capitalization and remove the punctuation and "and".